SU-HSDO/suhumsci
# News item node detail page

# READY FOR REVIEW

## Summary
Add styling for the node detail page, as described in 

## Need Review By (Date)
_asap_

## Urgency
_high_

## Steps to Test
(pairing through this might be easier, but...)
1. Create a news item node, with content populated in all of the fields (including categories).
2. Customize the node's layout to use layout-builder, if it isn't using it already.
3. Using layout-builder, with the 3-column-w-image layout, populate the fields as shown below:
![image](https://user-images.githubusercontent.com/1256329/73216556-31e5e700-4124-11ea-8089-2c4415c90ec7.png)
4. Confirm that it behaves like the design at all breakpoints
5. Extra credit: try various content iterations, like:
    - Some fields have no content
    - No sidebar, right-sidebar-only, left-sidebar-only, etc.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
